diff --git a/build.gradle b/build.gradle
index 4e43cc9..5ba05d5 100644
--- a/build.gradle
+++ b/build.gradle
@@ -73,7 +73,7 @@ shadowJar {
 dependencies {
     testImplementation platform('org.junit:junit-bom:5.10.0')
     testImplementation 'org.junit.jupiter:junit-jupiter'
-    compileOnly("com.hypixel.hytale:Server:0.5.0")
+    compileOnly("com.hypixel.hytale:Server:0.6.7")
 
 
     embed('com.vdurmont:emoji-java:5.1.1')
-- 
2.54.0

